docs/tutorial.rst

```diff
@@ -115,6 +114,10 @@ Let us fetch our new collection of tasks:
     }
 
 
+Keep a note of the ``ETag`` and of the ``last_modified`` values returned (here
+respectively ``"1434641474977"`` and ``"1434641515332"``), we'll need them for
```

Comment:
I would have expected them to be equal. Created an issue, discussion happens there: https://github.com/mozilla-services/cliquet/issues/356